entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Note that this file is for the exclusive use of the Dockerfile as an ENTRYPOINT
+set -e
+
+# Remove a potentially pre-existing server.pid for Rails.
+rm -f /code/tmp/pids/server.pid
```

Comment:
I had same problem in SS; why we have a problem with this file when we run Limber in Docker but not in native? What is different?

Reply:
I didn't have a problem.  I just followed the advice of the official Docker documentation: https://docs.docker.com/samples/rails/

Reply:
Looks good, the problem with this file was the reason why I had to start using exec in the entrypoint, so a docker-compose down would stop all services and remove this file; but this didn't work if the user kills with control-c. With this we cover all.